piston/configuration/validators/prompt_validator.py

```diff
@@ -0,0 +1,63 @@
+from piston.configuration.choose_config import choose_config
+from piston.configuration.validators.validator_base import Validator
+from piston.utils.constants import CONSOLE, Configuration
+
+
+class PromptStartValidator(Validator):
+    """Validates the prompt_start config variable."""
+
+    def __init__(self, prompt: str) -> None:
+        self.prompt = prompt
+        self.default_prompt = Configuration.default_configuration["prompt_start"]
+        super().__init__(self.prompt, self.default_prompt, "prompt_start")
+
+    def validate_prompt(self) -> bool:
+        """Validates prompt start."""
+        if not self.validate_type():
+            return False
+
+        # Make sure there are no new lines in the prompt
+        if "\n" in self.prompt or "\r" in self.prompt:
+            return False
+
+        return True
+
+    def fix_prompt(self) -> str:
+        """Fixes the prompt_start configuration variables."""
+        if self.validate_prompt():
+            return choose_config(self.prompt)
+        CONSOLE.print(
+            f"""[red]Prompt start invalid. "{self.prompt}" contains a new \
+                line character. Using default prompt start instead.[/red]"""
```

Comment:
- Personal preference, but I don't like the use `/` here, you can just break it into two strings and python automatically connects them into one.
- Say your `prompt_start` is `>>\n>` then it would be displayed with the newline in the error, a way to escape this is to use `repr()`

This is how the above looks

![Screenshot from 2021-05-25 10-03-28](https://user-images.githubusercontent.com/69356296/119439689-83b27780-bd40-11eb-8204-a629c45892a2.png)


```suggestion
            f"[red]Prompt start invalid. {repr(self.prompt)} contains a new"
            "line character. Using default prompt start instead.[/red]"
```

Do the same on `PromptContinuationValidator`